crates/bevy_window/src/lib.rs

```diff
@@ -118,6 +118,16 @@ impl Plugin for WindowPlugin {
         if settings.close_when_requested {
             app.add_system(close_when_requested);
         }
+
+        #[cfg(debug_assertions)]
+        {
```

Comment:
nit: `#[]` attributes work on statements, so no need to wrap `app.add_system(...)` into a block

Reply:
Whoops. I had a statement outside the system before.